hyperdome_client/hyperdome_client.py

```diff
@@ -204,9 +214,9 @@ def send_message(self):
                 self.chat_history.append("You: " + message)
                 self.on_history_added()
                 if self.server.is_therapist: # needs auth
-                    session.post(f"{self.server.url}/message_from_therapist",headers={"username":self.server.username, "password":self.server.password,"message":message})
+                    self.session.post(f"{self.server.url}/message_from_therapist",headers={"username":self.server.username, "password":self.server.password,"message":message})
```

Comment:
```suggestion
                    self.session.post(f"{self.server.url}/message_from_therapist", headers={"username":self.server.username, "password":self.server.password, "message":message})
```
flake8 compliant. Doesn't matter too much.